lib/action_controller/serialization.rb

```diff
@@ -30,7 +30,7 @@ def get_serializer(resource, options = {})
         serializable_resource.serialization_scope ||= serialization_scope
         serializable_resource.serialization_scope_name = _serialization_scope
         begin
-          serializable_resource.adapter
+          serializable_resource.tap(&:adapter)
```

Comment:
If specs don't break, it's a regression in the specs, since it was added in response to behavior https://github.com/rails-api/active_model_serializers/pull/962#issuecomment-114204858

> There was a crash when rendering arrays that don't consist of "serializables" classes.